.gitignore

```diff
@@ -1,14 +1,23 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
 node_modules
+/.pnp
+.pnp.js
```

Comment:
just curious, what file is this?

Reply:
im not sure. I copied all the .gitignore from frontend and server to the root to have only one .gitignore. I think that came from the .gitignore in the server